README.md

```diff
@@ -52,46 +86,92 @@ The GOBL to XInvoice tool also includes a command line helper. You can it instal
 go install ./cmd/gobl.xinvoice
 ```
 
-Usage is very straightforward:
+Usage is very straightforward, with the only flag being the preferred output format: `xrechnung-cii`, `xrechnung-ubl`, `zugferd` or `facturx`, to be used in case of conversion from GOBL. It is also possible to save the output to a file by adding a path as its second argument:
 
 ```bash
-gobl.xinvoice convert ./test/data/invoice-de-de.json
+# XInvoice to GOBL
+gobl.xinvoice convert ./test/data/invoice-de-de.xml
+
+# XInvoice to GOBL (with output file)
+gobl.xinvoice convert ./test/data/invoice-de-de.xml ./test/data/out/invoice-de-de-gobl.json
+
+# GOBL to XRechnung CII
+gobl.xinvoice convert ./test/data/invoice-de-de.json --format xrechnung-cii
+
+# GOBL to Factur-X CII
+gobl.xinvoice convert ./test/data/invoice-de-de.json --format facturx
 ```
 
 ## Testing
 
 ### testify
 
 The library uses testify for testing. To run the tests you can use the command:
-```
+```bash
 go test
 ```
 
-### Github action
+## Development
 
-There is a additional validator for XRechnung that is provided as a github action. You can use it to make sure that the created documents are valid. To run it go to:
-```
-https://github.com/invopop/gobl.xinvoice/actions/workflows/xrechnung_validator.yaml
-```
-Select a branch on which the file you want to test is on. Provide it with a file path to a file in the repository and click Run Workflow
+### Limitations
 
-## Development
+There are some limitations in the current conversion process to GOBL.
+
+#### CII
+
+1. GOBL does not currently support additional embedded documents, so the AdditionalReferencedDocument field (BG-24 in EN 16931) is not supported and lost in the conversion.
+2. Payment advances do not include their own tax rate, they use the global tax rate of the invoice.
+3. The field TotalPrepaidAmount (BT-113) in CII is not directly mapped to GOBL, so payment advances must be included in the SpecifiedAdvancePayment field in CII, or they will be lost in conversion.
+4. Fields BusinessProcessSpecifiedDocumentContextParameter (BT-23) and GuidelineSpecifiedDocumentContextParameter (BT-24) in CII are not supported and lost in the conversion.
+5. The fields ReceivableSpecifiedTradeAccountingAccount (BT-133) and DesignatedProductClassification (BT-158) are added as a note to the line, with the type code as the key.
+6. The GlobalID (BT-29, BT-46, BT-60,BT-157) fields are currently not directly mapped to their [codelist](https://docs.peppol.eu/poacc/billing/3.0/codelist/ICD/) ISO 6523 ICD, with their reference code being mapped to the Label.
```

Comment:
I think this is supported now with the extensions in GOBL's `org.Identity`.

Reply:
Just updated the converters [CII](https://github.com/invopop/gobl.cii/pull/5) and [UBL](https://github.com/invopop/gobl.ubl/pull/4) to handle that better, alongside the Address.LineOne update